src/components/SocialMediaSharing.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import { Grid } from '@material-ui/core';
+import {
+  EmailIcon,
+  EmailShareButton,
+  FacebookIcon,
+  FacebookShareButton,
+  LinkedinIcon,
+  LinkedinShareButton,
+  RedditIcon,
+  RedditShareButton,
+  TwitterIcon,
+  TwitterShareButton,
+} from 'react-share';
+
+interface SocialMediaSharingProps {
+  resourceTitle: string;
+  url: string;
+}
+
+const SocialMediaSharing: FC<SocialMediaSharingProps> = ({ resourceTitle, url }) => {
+  return (
+    <Grid container spacing={1}>
+      <Grid item>
+        <FacebookShareButton quote={resourceTitle} url={url}>
+          <FacebookIcon size={32} round />
+        </FacebookShareButton>
+      </Grid>
+      <Grid item>
+        <TwitterShareButton url={url} title={resourceTitle}>
+          <TwitterIcon size={32} round />
+        </TwitterShareButton>
+      </Grid>
+      <Grid item>
+        <LinkedinShareButton url={url}>
+          <LinkedinIcon size={32} round />
+        </LinkedinShareButton>
+      </Grid>
+      <Grid item>
+        <RedditShareButton url={url} title={resourceTitle} windowWidth={660} windowHeight={460}>
```

Comment:
fixed size kan være skummelt når man skal ha både for desktop og mobil, men hvis det er testet for begge er det vel greit nok

Reply:
Fikset